mesh/mesh.go

```diff
@@ -51,43 +58,44 @@ type Mesh struct {
 	done          chan struct{}
 }
 
-func NewPersistentMesh(path string, rewardConfig RewardConfig, mesh MeshValidator, state StateUpdater, logger log.Log) *Mesh {
+func NewPersistentMesh(path string, rewardConfig Config, mesh MeshValidator, state StateUpdater, logger log.Log) *Mesh {
 	//todo add boot from disk
 	ll := &Mesh{
-		Log:          logger,
-		tortoise:     mesh,
-		state:        state,
-		done:         make(chan struct{}),
-		mdb:          NewPersistentMeshDB(path, logger),
-		rewardConfig: rewardConfig,
+		Log:      logger,
+		tortoise: mesh,
+		state:    state,
+		done:     make(chan struct{}),
+		mdb:      NewPersistentMeshDB(path, logger),
+		config:   rewardConfig,
 	}
 
 	return ll
 }
 
-func NewMemMesh(rewardConfig RewardConfig, mesh MeshValidator, state StateUpdater, logger log.Log) *Mesh {
+func NewMemMesh(rewardConfig Config, mesh MeshValidator, state StateUpdater, logger log.Log) *Mesh {
```

Comment:
same

Reply:
I've removed config that is not reward related so it's not really relevant at the moment